src/operator/tensor/matrix_op.cc

```diff
@@ -354,6 +354,60 @@ NNVM_REGISTER_OP(dot)
 
     dot(x,y)[i,j,a,b] = sum(x[i,j,:]*y[:,a,b])
 
+  Example::
+
+    x = [[[  0.,   1.,   2.,   3.],
+	  [  4.,   5.,   6.,   7.],
+	  [  8.,   9.,  10.,  11.]],
+
+	 [[ 12.,  13.,  14.,  15.],
```

Comment:
See numpy doc: https://docs.scipy.org/doc/numpy-1.12.0/reference/generated/numpy.dot.html
we don't want to flood users with meaning less numbers